public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -92,6 +92,39 @@ mciModule.controller('PerformanceDiscoveryCtrl', function (
     vm.updateData(true);
   }
 
+  $scope.changeMetric = function () {
+    const fromVersion = vm.fromSelect.selected
```

Comment:
I don't think any of these constants are being used here.

Reply:
deleted the unused ones